types/storybook-readme/index.d.ts

```diff
@@ -0,0 +1,48 @@
+// Type definitions for storybook-readme 4.0
+// Project: https://github.com/tuchk4/storybook-readme
+// Definitions by: Taeheon Kim  <https://github.com/lonyele>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.9
```

Comment:
This should be the minimum version that works -- it looks like `// TypeScript Version: 2.3` works just as well.

Reply:
oh I checked "DefinitelyTyped/node_modules/types-publisher/bin/checkTypeScriptVersions"
I consoled pkg, dep and found react version that DefinitelyTyped is using typeScriptVersion: '2.8'   This is why... interesting